scrapers/Trans500.yml

```diff
@@ -0,0 +1,19 @@
+name: trans500
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - trans500.com
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="col-sm-12"]/h2
```

Comment:
this returns an extra result `Highest Ratest Videos`
using 
```yaml
      Title: //div[@class="col-sm-12"]/p/../h2/text()
```
or 
```yaml
      Title: (//div[@class="col-sm-12"])[1]/h2
```
seems to fix that

Reply:
Okay, will try both and test on a few scenes.